test/whitePaperModel/mining/borrowMiningTests.spec.ts

```diff
@@ -1188,6 +1195,238 @@ contract("borrowMiningTests", async (accounts) => {
+                                borrowAmount
+                            );
+                            console.log("------------------ user 2 borrows -------------------");
+                            // Deposit an extra token to create a new rate check point
+                            await savingAccount.fastForward(1000);
+                            await savingAccount.deposit(erc20DAI.address, new BN(10), {
+                                from: user1,
+                            });
+
+                            // 4. Claim the minted tokens
+                            // fastforward
+                            await savingAccount.fastForward(100000);
+                            // Deposit an extra tokens to create a new rate check point
+                            await savingAccount.deposit(erc20DAI.address, new BN(10), {
+                                from: user1,
+                            });
+
+                            const balFINUser2 = await erc20FIN.balanceOf(user2);
+                            console.log("balFINUser2", balFINUser2.toString());
+                            const balFINUser1 = await erc20FIN.balanceOf(user1);
+                            console.log("balFINUser1", balFINUser1.toString());
+
+                            // FIN balance before claim
+                            // User 1
+                            const claimDepositUser1 = new BN(
+                                await savingAccount.claimForToken.call(addressDAI, {
+                                    from: user1,
+                                })
+                            );
+                            console.log("claimDepositUser1", claimDepositUser1.toString());
+
+                            const claimDAIUser1 = new BN(
+                                await savingAccount.claimForToken.call(addressDAI, {
+                                    from: user1,
+                                })
+                            );
+                            console.log("claimDAIUser1", claimDAIUser1.toString());
+
+                            const TotalclaimableAmountUser1 = new BN(
+                                await savingAccount.claim.call({
+                                    from: user1,
+                                })
+                            );
+                            console.log(
+                                "TotalclaimableAmountUser1",
+                                TotalclaimableAmountUser1.toString()
+                            );
+
+                            // User 2
+                            const claimDAIUser2 = new BN(
+                                await savingAccount.claimForToken.call(addressDAI, {
+                                    from: user2,
+                                })
+                            );
+                            console.log("claimDAIUser2", claimDAIUser2.toString());
+
+                            const TotalclaimableAmountUser2 = new BN(
```

Comment:
expect `TotalclaimableAmountUser2`  to be greater than zero.

Reply:
Have added this and also changed to camelCase